cfgov/jinja2/v1/events/_macros.html

```diff
@@ -222,12 +222,12 @@ <h2 class="event-venue_heading">
                 {% for speaker in bound.speakers.value %}
                     {%- if speaker.bound_blocks.url.render() -%}
                     <a href="{{ speaker.bound_blocks.url.render() }}">
-                    {{ speaker.bound_blocks.name.render() }}
+                    {{- speaker.bound_blocks.name.render() -}}
                     </a>
                     {%- else -%}
-                    {{ speaker.bound_blocks.name.render() }}
+                    {{- speaker.bound_blocks.name.render() -}}
                     {%- endif -%}
-                    {{ ';' if loop.index < loop.length else '' }}
+                    {{ '; ' if loop.index < loop.length else '' }}
```

Comment:
Sorry, I'm stripping it from speaker names and then adding a space after semicolons.